stock_available/data/ir_config_parameter.xml

```diff
@@ -0,0 +1,6 @@
+<odoo noupdate="1">
+    <record id="default_stock_available_mrp_based_on" model="ir.config_parameter">
+        <field name="key">stock_available.stock_available_mrp_based_on</field>
```

Comment:
Why having this parameter for `mrp`?

Reply:
Basically, this is also a field for configuration, and it is only used for the MRP bridge module.
Previously, it was with get_values and set_values with default value.
We have changed it to the config_parameter and created the data for it to support the default value.

It is also a question of whether it should remain in this module or be moved to the MRP bridge module.

Please suggest the proper approach, and we will do the same.

Reply:
But it's the same as the module installation: this is for modules outside this one, so they should be on the corresponding module (`stock_available_mrp`).

Reply:
@pedrobaeza an opinion regarding this question 

https://github.com/BizzAppDev-Systems/stock-logistics-availability/pull/2/files#r1661123966
> In v16, the ICP was in the stock_available module. Now, it has been moved to the stock_available_mrp module. This migration script will be helpful for those who already have the v16 version installed for them. Question is: For the migration script, should we keep the old name and move it to this module or change the name?